charts/ratify/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: ratify
 description: A Helm chart for Ratify
-version: 1.0.0
+version: 1.0.1
```

Comment:
Hi @hbahadorzadeh , we can take care of the chart update on the next release. I think it is okay to keep 1.0.0 as is. The latest version should be 1.0.0, https://github.com/deislabs/ratify/releases/tag/v1.0.0-alpha.2

Reply:
This has been addressed by PR https://github.com/deislabs/ratify/pull/378,